setup/plugin/config/operator.yaml

```diff
@@ -0,0 +1,15 @@
+
+# Operator ECDSA address
+operator_address: 0xD5A0359da7B310917d7760385516B2426E86ab7f
+
+# AVS contract addresses
+avs_registry_coordinator_address: 0x0069A298e68c09B047E5447b3b762E42114a99a2
+operator_state_retriever_address: 0x8D0b27Df027bc5C41855Da352Da4B5B2C406c1F0
+
+# AVS network RPCs
+eth_rpc_url: https://ethereum-holesky-rpc.publicnode.com
+eth_ws_url: wss://ethereum-holesky-rpc.publicnode.com
+
+# EigenLayer ECDSA and BLS private key paths
+ecdsa_private_key_store_path: /near-sffl/config/keys/ecdsa.json
+bls_private_key_store_path: /near-sffl/config/keys/bls.json
```

Comment:
Not sure about this one, but shouldn't these paths be relative (`./`) rather than absolute (`/`)?